includes/admin/class-wc-payments-admin.php

```diff
@@ -238,8 +238,16 @@ public function register_payments_scripts() {
 		$error_message = get_transient( WC_Payments_Account::ERROR_MESSAGE_TRANSIENT );
 		delete_transient( WC_Payments_Account::ERROR_MESSAGE_TRANSIENT );
 
+		// The country is identified from business address, which merchant provided after setting up WooCommerce.
+		// The default value is taken from https://github.com/woocommerce/woocommerce/issues/29499.
+		$business_country = explode( ':', get_option( 'woocommerce_default_country', 'US:CA' ), 2 )[0];
```

Comment:
Wondering if we should utilize `wc_get_base_location()` instead, as it also utilizes a hook for extensibility.

Reply:
Looks like a more reliable option to me, but I am not completely sure about this, what do you think @Automattic/gamma?

Reply:
With a fresh head, I believe we should rely on the function, as the functionality is part of WooCommerce core and we'll be covered from any possible changes there. Updated in a4cf229d7f58457faf1d089e3d1b97634945792f